### DISCUSSION TITLE
提高生育率!!!

### DISCUSSION THREAD

**Discussion by henrywen547** [1101-GS4719-C/Python-Community-GS4719-C#522] [General]
少子化很早開始就是日本的問題，夫妻無法一人辭掉工作去照顧小孩，為了養家只能是雙薪家庭。
日本政府則推廣育兒科技，像是以AI辨識嬰兒各種哭聲代表的意義等等，希望能減輕養小孩的負擔。
如果台灣的薪資問題有改善，科技是否有助提高生育率呢?
現在多數人顧小孩是找保母或找爺爺奶奶，我個人是覺得育兒科技在「讓家庭關係變好」這塊應該有更好的效果啦
(畢竟有了錢大家就比較想生了吧)
[生育率太低要怎樣減低育兒負擔？日本政府推廣「育兒科技」](https://udn.com/news/story/6897/6148131)

**Comment by cordydeng** [comment]
但我覺得根本的問題在於經濟方面 , 大家顧慮的問題還是在於能不能在生活過下去的前提下養活孩子

**Comment by hapiwangy** [comment]
少子化問題已經在世界上很多高度發展的社會已經造成許多影響，提高生育率也許是最直接的解決方法，但很多其他因素讓這個解決方法已經不那麼的合適

**Comment by leon10456** [comment]
我覺得少子化還是因為經濟問題佔大宗 只要有錢 就能把小孩送去育嬰中心 學才藝等等的
尤其是現在大部分人都不想因為生小孩而降低生活品質 還是給育嬰津貼 生二胎以上保證公托名額比較實在

**Comment by Sazmafuji** [comment]
生育率問題會影響未來的經濟方向，而如果現在就因為經濟問題而產生少子化，未來想必會更慘，現在需改變大部分人想法，以防此惡循環產生。

**Comment by HuanYuuuuuu** [comment]
感覺在台灣也需要想一下該如何應對少子化的問題
像我自己如果沒有到有足夠穩定的經濟能力就不會想要生小孩
不然只是讓小孩受苦而已...

**Comment by Yuhsin000** [comment]
我覺得就算薪資水準有改善，AI 對減輕養小孩的負擔的影響也有限，且小孩的成長是非常需要爸媽的陪伴的，爸媽不只要負責小孩的生理需求，還有注意、輔助小孩的發展，AI 很難做到。

**Comment by xkllkx** [comment]
感覺經濟狀況才是主因，就算有科技可以減輕養小孩負擔，但照顧小孩的工作還是需要人做，會一根蠟燭兩頭燒。

**Comment by awkward-willy** [comment]
不論是否改善經濟，若一昧的將照顧小孩的工作推給保母、托嬰中心，甚至是 AI、機器人
可能只會造成更多的親子關係疏離的問題
或許該怎麼讓父母空出時間與小孩培養關係並真正與小孩有身心層面的溝通才是我們需要思考的要點

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望AI可以改善我們的生活。

**Comment by ghost** [comment]
能不能照顧是一回事，最實際的問題永遠是金錢方面

**Comment by Jay-722** [comment]
現在的薪水，養孩子都難了，還用高科技來養

**Comment by dayu1004** [comment]
工作內容越耗精力的人越不願意生小孩，高知識份子也怕自己交不好小孩不敢生。所以我覺得僅靠AI帶小孩好像不太會對於生育有太大的誘因

**Comment by leon10456** [comment]
主要還是經濟問題吧..

**Comment by HaoFish123** [comment]
希望科技可以改善未來許多社會問題

**Comment by HuanYuuuuuu** [comment]
感覺嬰兒想哭就只有餓了或是大便尿尿這幾個原因吧
而且最主要是年輕人根本沒有經濟能力去負擔再養一個小孩
